src/yb/master/catalog_manager.cc

```diff
@@ -4046,6 +4046,14 @@ Status CatalogManager::GetBackfillJobs(
   return Status::OK();
 }
 
+Status CatalogManager::FlushSysCatalog(
+    const FlushSysCatalogRequestPB* req,
+    FlushSysCatalogResponsePB* resp,
+    rpc::RpcContext* rpc) {
+  RETURN_NOT_OK(sys_catalog_->tablet_peer()->shared_tablet()->Flush(tablet::FlushMode::kSync));
+  return Status::OK();
+}
```

Comment:
we need to return/populate an error in the response in case this flush operation fails. see GetBackfillJobs just above this for an example:
```
if (indexed_table == nullptr) {
    Status s = STATUS(NotFound, "Requested table $0 does not exist", table_id.ShortDebugString());
    return SetupError(resp->mutable_error(), MasterErrorPB::OBJECT_NOT_FOUND, s);
  }
```

Reply:
will return_not_ok populate the error from Flush(tablet::FlushMode::kSync) back as a bad status?

Reply:
@Tigojiang yes!